File: Truck-Driver-Tracker API/Truck Drive.DAL/Entities/Truck.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Threading.Tasks;

namespace Truck_Drive.DAL.Entities
{
    public class Truck
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public int Id { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }
        public long SiteId { get; set; }
        public long AssetId { get; set; }
        public string RegistrationNumber { get; set; }
        public DateTime LastPositionTimestamp { get; set; }
        public double Latitude { get; set; }
        public double Longitute { get; set; }
      
        public string CurrentAddress { get; set; }
    }
}

File: Truck-Driver-Tracker-UI/Lafarge GPS UI/src/app/_interface/truck.ts
export interface Driver {
    id: number;
    createdAt: Date;
    updatedAt: Date;
    siteId: number;
    driverId: number;
    name: string;
    employeeNumber: string;
  }

  
  export interface Truck {
    id: number;
    createdAt: Date;
    updatedAt: Date;
    siteId: number;
    assetId: number;
    registrationNumber: string;
    lastPositionTimestamp: Date;
    latitude: number;
    longitude: number;
    currentAddress: string;
    
  }
  
 